seisma/api/resourses/v1/cases.py

```diff
@@ -113,6 +113,38 @@ def add_case_to_job(job_name, case_name):
         return make_result(case), statuses.CREATED
 
 
+@resource.route(
+    '/jobs/<string:job_name>/cases/<string:case_name>',
+    methods=['PUT'],
+    schema='case.post_put.json',
+)
+def update_case_in_job(job_name, case_name):
+    """
+    .. http:put:: /jobs/(string:job_name)/cases/(string:case_name)
+
+    Update case in job, statistic about case stored separated of case data.
+
+    :jsonparam string description: about case. it's can be docstring for example
+
+    :return: Case Resource
+    """
+    job = db.Job.get_by_name(job_name)
+
+    if job:
+        case = db.Case.query.filter_by(job_id=job.id, name=case_name).first()
+
+        if case:
+
+            json = flask.request.get_json()
+
+            data = {
+                'description': json.get('description', ''),
+            }
+            case = case.update(**data)
+
+        return make_result(case), statuses.OK
```

Comment:
It looks like `return` should be inside `if case:` statement.

Reply:
Yes, fixed.